dev/bots/suite_runners/run_verify_binaries_codesigned_tests.dart

```diff
@@ -89,16 +89,10 @@ List<String> binariesWithoutEntitlements(String flutterRoot) {
     'artifacts/engine/darwin-x64/libtessellator.dylib',
     'artifacts/engine/ios-profile/Flutter.xcframework/ios-arm64/Flutter.framework/Flutter',
     'artifacts/engine/ios-profile/Flutter.xcframework/ios-arm64_x86_64-simulator/Flutter.framework/Flutter',
-    'artifacts/engine/ios-profile/extension_safe/Flutter.xcframework/ios-arm64/Flutter.framework/Flutter',
```

Comment:
You'll need to re-add this until `extension_safe` is removed after a release or so.